python/hail/methods/impex.py

```diff
@@ -521,3 +529,135 @@ def import_fam(path, quant_pheno=False, delimiter=r'\\s+', missing='NA'):
+def import_gen(path, sample_file=None, tolerance=0.2, min_partitions=None, chromosome=None, reference_genome=None,
+               contig_recoding=None):
+    return Env.hc().import_gen(path, sample_file, tolerance, min_partitions, chromosome, reference_genome,
+                               contig_recoding).to_hail2()
+
+
+@handle_py4j
+@typecheck(paths=oneof(strlike, listof(strlike)),
+           key=oneof(strlike, listof(strlike)),
+           min_partitions=nullable(int),
+           impute=bool,
+           no_header=bool,
+           comment=nullable(strlike),
+           delimiter=strlike,
+           missing=strlike,
+           types=dictof(strlike, Type),
+           quote=nullable(char),
+           reference_genome=nullable(GenomeReference))
+def import_table(paths, key=[], min_partitions=None, impute=False, no_header=False,
+                 comment=None, delimiter="\t", missing="NA", types={}, quote=None, reference_genome=None):
+    return Env.hc().import_table(paths, key, min_partitions, impute, no_header, comment,
+                                 delimiter, missing, types, quote, reference_genome).to_hail2()
+
+
+@handle_py4j
+@typecheck(bed=strlike,
+           bim=strlike,
+           fam=strlike,
+           min_partitions=nullable(integral),
+           delimiter=strlike,
+           missing=strlike,
+           quant_pheno=bool,
+           a2_reference=bool,
+           reference_genome=nullable(GenomeReference),
+           contig_recoding=nullable(dictof(strlike, strlike)),
+           drop_chr0=bool)
+def import_plink(bed, bim, fam, min_partitions=None, delimiter='\\\\s+',
+                 missing='NA', quant_pheno=False, a2_reference=True, reference_genome=None,
+                 contig_recoding={'23': 'X', '24': 'Y', '25': 'X', '26': 'MT'}, drop_chr0=False):
+    return Env.hc().import_plink(bed, bim, fam, min_partitions, delimiter,
+                                 missing, quant_pheno, a2_reference, reference_genome, contig_recoding,
+                                 drop_chr0).to_hail2()
+
+
+@handle_py4j
+@typecheck(path=oneof(strlike, listof(strlike)),
+           drop_cols=bool,
+           drop_rows=bool)
+def read_matrix(path, drop_cols=False, drop_rows=False):
+    return Env.hc().read(path, drop_cols, drop_rows).to_hail2()
+
+
+@handle_py4j
+@typecheck(path=oneof(strlike, listof(strlike)))
+def get_vcf_metadata(path):
+    return Env.hc().get_vcf_metadata(path)
+
+
+@handle_py4j
+@typecheck(path=oneof(strlike, listof(strlike)),
+           force=bool,
+           force_bgz=bool,
+           header_file=nullable(strlike),
+           min_partitions=nullable(integral),
+           drop_samples=bool,
+           call_fields=oneof(strlike, listof(strlike)),
+           reference_genome=nullable(GenomeReference),
+           contig_recoding=nullable(dictof(strlike, strlike)))
+def import_vcf(path, force=False, force_bgz=False, header_file=None, min_partitions=None,
```

Comment:
I still think methods.import_vcf still reads strange, but I can get used to it.  But we're going to import of methods into toplevel scope when we switch to `import hail as hl`, right?  Then I'm OK.

Reply:
yes, this will be `hl.import_vcf`